tensorflow_addons/image/resampler_ops_test.py

```diff
@@ -193,62 +193,64 @@ def _test_op_backward_pass(self, on_gpu, dtype):
                         t, n, float_rtol=5e-5, float_atol=5e-5
                     )
 
-    def test_op_errors(self):
-        batch_size = 10
-        data_height = 9
-        data_width = 7
-        data_depth = 3
-        data_channels = 5
-        warp_width = 4
-        warp_height = 8
-
-        # Input data shape is not defined over a 2D grid, i.e. its shape is not like
-        # (batch_size, data_height, data_width, data_channels).
-        data_shape = (batch_size, data_height, data_width, data_depth, data_channels)
-        data = np.zeros(data_shape)
-        warp_shape = (batch_size, warp_height, warp_width, 2)
-        warp = np.zeros(warp_shape)
-
-        with self.assertRaisesRegexp(
-            tf.errors.UnimplementedError,
-            "Only bilinear interpolation is currently supported.",
-        ):
-            self.evaluate(resampler_ops.resampler(data, warp))
 
-        # Warp tensor must be at least a matrix, with shape [batch_size, 2].
-        data_shape = (batch_size, data_height, data_width, data_channels)
-        data = np.zeros(data_shape)
-        warp_shape = (batch_size,)
-        warp = np.zeros(warp_shape)
-
-        with self.assertRaisesRegexp(
-            tf.errors.InvalidArgumentError, "warp should be at least a matrix"
-        ):
-            self.evaluate(resampler_ops.resampler(data, warp))
-
-        # The batch size of the data and warp tensors must be the same.
-        data_shape = (batch_size, data_height, data_width, data_channels)
-        data = np.zeros(data_shape)
-        warp_shape = (batch_size + 1, warp_height, warp_width, 2)
-        warp = np.zeros(warp_shape)
-
-        with self.assertRaisesRegexp(
-            tf.errors.InvalidArgumentError, "Batch size of data and warp tensor"
-        ):
-            self.evaluate(resampler_ops.resampler(data, warp))
-
-        # The warp tensor must contain 2D coordinates, i.e. its shape last dimension
-        # must be 2.
-        data_shape = (batch_size, data_height, data_width, data_channels)
-        data = np.zeros(data_shape)
-        warp_shape = (batch_size, warp_height, warp_width, 3)
-        warp = np.zeros(warp_shape)
-
-        with self.assertRaisesRegexp(
-            tf.errors.UnimplementedError,
-            "Only bilinear interpolation is supported, warping",
-        ):
-            self.evaluate(resampler_ops.resampler(data, warp))
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_op_errors():
```

Comment:
LGTM. Thanks, @gabrieldemarmiesse. 
Just curious about how we decide what tests should be moved out? Could you please also share the thoughts?  The [PR](https://github.com/tensorflow/addons/pull/1438/files) is a good example of my question: can we also move this? https://github.com/tensorflow/addons/blob/81479e2fcfee09541dae57ccfd9c4e43fe0ff311/tensorflow_addons/layers/normalizations_test.py#L292

Reply:
Sorry I forgot to link the issue: #1328

We should move all the tests, including the one you're mentioning. But I'm doing it one at a time to make pull requests easier to review. You wouldn't want to review 2000+ lines of code at once :)

Reply:
That makes sense. Thanks @gabrieldemarmiesse.